packages/thirdweb/src/extensions/farcaster/signers.ts

```diff
@@ -0,0 +1,25 @@
+import { fromBytes } from "../../utils/encoding/from-bytes.js";
+import type { Hex } from "../../utils/encoding/hex.js";
+
+export type Ed25519Keypair = {
+  publicKey: Hex;
+  privateKey: Hex;
+};
+
+/**
+ * Generates an Ed25519 keypair to be used as an account signer.
+ * @returns A promise resolving to the generated keypair.
+ * @example
+ * ```
+ * createSigner()
+ * ```
+ */
+export async function createSigner(): Promise<Ed25519Keypair> {
+  const ed = await import("@noble/ed25519");
```

Comment:
this is required every time we use this - it makes sense to just import it statically

Reply:
Missed this one, will fix